test/sql/lineage/simple_aggr_lineage/test_simple_aggr_lineage.test

```diff
@@ -0,0 +1,308 @@
+# name: test/sql/lineage/simple_aggr_lineage/test_simple_aggr_lineage.test
+# description: Test Simple Aggregate Lineage
+# group: [simple_aggr_lineage]
+
+
+statement ok
+PRAGMA explain_output = PHYSICAL_ONLY;
+
+statement ok
+CREATE TABLE t1(i INTEGER);
+
+statement ok
+INSERT INTO t1 SELECT i FROM range(0,5) tbl(i);
+
+statement ok
+PRAGMA trace_lineage = "ON";
+
+query I
+select COUNT(*) from t1
+----
+5
+
+statement ok
+PRAGMA trace_lineage = "OFF";
+
+query II
+SELECT * FROM queries_list;
+----
+9	select COUNT(*) from t1
+11	PRAGMA trace_lineage = "OFF";
+
+
+query IIII
+select rowid, chunk_id, range_start, range_end from SEQ_SCAN_9_0_range;
```

Comment:
Did you run this on the master branch? The schema is slightly different https://github.com/reallocf/duckdb/commit/6cc9c3972c031b36ea727c237c131ffb59cbb64a

Reply:
nope not yet was going through the code. Will check them out now.